<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Items Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="businessWindow_edit_dialog_nameLine">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="priceLabel">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Price</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="businessWindow_edit_dialog_priceLine">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addItemButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateItemButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeItemButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAllItemButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove All Items</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="moveUpButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>▲</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveDownButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>▼</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QTableWidget" name="businessWindow_edit_dialog_table">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
